tests/functional/aws-node-sdk/test/legacy/tests.js

```diff
@@ -254,11 +258,11 @@ describe('aws-node-sdk test suite as registered user', function testSuite() {
             MultipartUpload: {
                 Parts: [
                     {
-                        ETag: `"${calculatedFirstPartHash}"`,
+                        ETag: calculatedFirstPartHash,
```

Comment:
As per our conversation, indeed these quotes are unnecessary to create the MPU JSON structure.